### PR TITLE
show name from microsoft id

### PR DESCRIPTION
**Pull request by yonsweng**

### CODE REVIEW

**Comment by yonsweng** [comment]
logo192 is copied cuz of a error log at the browser.

**Comment by yonsweng** [comment]
Can we maintain (one-app one-url one-js-file) policy like this pr? Or Is it not reasonable?

File: src/components/chat.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+import {FormControl, InputGroup, Container, Nav, Navbar, NavDropdown} from "react-bootstrap";
+
+import logo192 from '../assets/images/logo192.png'
+
+
+class CustomNavbar extends React.Component {
```
Comment (YoohaBae) [inline src/components/chat.js:8]: I actually separated the CustomNavbar from the chat page as the navbar can be reused multiple times. I think we should separate the concept of components and pages. It seams reasonable that components are objects that are reused multiple times such as a navbar and pages are only used once (in this case, chat page).

**Review by YoohaBae — CHANGES_REQUESTED** [review]

**Comment by YoohaBae** [comment]
> Can we maintain (one-app one-url one-js-file) policy like this pr? Or Is it not reasonable?

I think this is reasonable in the sense that the one-js-file is a page file. Multiple component js files should be allowed and the page js file can just import the component js files.

**Comment by yonsweng** [comment]
> > Can we maintain (one-app one-url one-js-file) policy like this pr? Or Is it not reasonable?
> 
> 
> 
> I think this is reasonable in the sense that the one-js-file is a page file. Multiple component js files should be allowed and the page js file can just import the component js files.

Agree. By the way, I think components that are shared over multi pages should be in the different js files.

File: src/components/auth.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  const code = searchParams.get('code');
+  const sessionState = searchParams.get('session_state');
+
+  // axios.post('http://localhost:8000/apps/user/v1/auth/', {
+  axios.post('https://talk.gausslabs.ai/api/apps/user/v1/auth/', {
+    code: code,
+    session_state: sessionState,
+  })
+  .then(function (response) {
+    var loginMessage = document.getElementById('login-message');
```
Comment (YoohaBae) [inline src/components/auth.js:16]: I think using let is a better convention for defining variables.
var allows to declare variables with the same name multiple times while let does not allow it.
Using let can help prevent overwriting important variables.
I wasn't aware of this but maybe we should consider let in the future.

**Review by YoohaBae — APPROVED** [review]
lgtm

**Pull request merged**